CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# 1.0.0 (2020-11-20)
+
+The release candidate (v1.0.0-rc.1) has been selected as a final release of v1.0.0.
```

Comment:
We should add to the line  "..with the following change"

Reply:
Addressed